src/Microsoft.Fx.Portability.Reports.Html/Resources/_PortabilityReport.cshtml

```diff
@@ -257,13 +218,13 @@
                         {
                             <td>@nugetPackage.AssemblyInfo</td>
                         }
-                      
+
```

Comment:
Why the new lines?

Reply:
Each of these is a line of spaces being replaced with a single newline.